src/component/OperationButtons.js

```diff
@@ -0,0 +1,9 @@
+import { operators } from "../const";
+import Button from "./Button";
+
+export default function operationButtons() {
+  const operationButtons = operators.map((v) => (
```

Comment:
1. `(v)`가 무엇인지 더 명시적으로 표기해주는게 좋겠어요!
2. `operationButtons()` 컴포넌트 명이랑 겹치지 않을까요?
3. 라인이 길지 않은데 굳이 별도로 분리할 필요는 없지 않을까요?

Reply:
> 1. (v)가 무엇인지 더 명시적으로 표기해주는게 좋겠어요!
> 2. operationButtons() 컴포넌트 명이랑 겹치지 않을까요?

```suggestion
  const operations = operators.map((operator) => (
```
operators 배열에서 가져오는 value 이므로 `v` -> `operator`로 변경했습니다.
컴포넌트명도 camel Case로 작성되어 있었네요 😅 Pascal Case 로 변경했습니다!

`operationButtons()` -> `OperationButtons()`


> 3. 라인이 길지 않은데 굳이 별도로 분리할 필요는 없지 않을까요?

말씀하시는 대상 라인이 map 함수를 담는 `const operationButtons` 일까요,
아니면 `OperationButtons.js`를 말씀하시는 걸까요? 👀

전자라면 return 구문 안에서 최대한 가독성있게끔 작성하려고 했던 것 같고,
후자라면 작성 당시 Calculator 하위에 DigitButtons와 ModifierButtons, OperationButtons 가
일관성있게 모듈화되어야 한다고 생각했던 것 같습니다!

혹시 성능상의 차이가 있을까요? 🫢

Reply:
1, 2번은 해결된 것 같고 3번만 답변드리자면 모듈화하기에는 코드 볼륨이 적다보니 오히려 불편할 수 있어서 그런 의견을 드렸답니다 :)
또한 이런 상황이라면 성능차이는 없을 것 같네요!